flink-connector-gcp-pubsub-e2e-tests/src/test/java/org/apache/flink/connector/gcp/pubsub/sink/util/PubsubHelper.java

```diff
@@ -36,27 +40,50 @@
 import com.google.pubsub.v1.ReceivedMessage;
 import com.google.pubsub.v1.Topic;
 import com.google.pubsub.v1.TopicName;
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import java.io.IOException;
+import java.time.Duration;
 import java.util.List;
+import java.util.concurrent.TimeUnit;
 import java.util.stream.Collectors;
 
 /** A helper class to make managing the testing topics a bit easier. */
 public class PubsubHelper {
 
     private static final Logger LOG = LoggerFactory.getLogger(PubsubHelper.class);
 
+    private static final Duration SHUTDOWN_TIMEOUT = Duration.ofSeconds(5);
+
+    private ManagedChannel channel;
+
     private TransportChannelProvider channelProvider;
 
     private TopicAdminClient topicClient;
+
     private SubscriptionAdminClient subscriptionAdminClient;
 
+    public PubsubHelper(String endpoint) {
+        channel = ManagedChannelBuilder.forTarget(endpoint).usePlaintext().build();
+        channelProvider =
+                FixedTransportChannelProvider.create(GrpcTransportChannel.create(channel));
```

Comment:
`checkNonNull(channel)`
`checkNonNull(channelProvider)` ?

Reply:
I am not following we are creating channel and channelProvider here, why do we need to check?